dd-java-agent/instrumentation/vertx-web-3.4/src/main/java/datadog/trace/instrumentation/vertx_3_4/server/HttpServerResponseInstrumentation.java

```diff
@@ -0,0 +1,75 @@
+package datadog.trace.instrumentation.vertx_3_4.server;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.HierarchyMatchers.implementsInterface;
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+import static datadog.trace.instrumentation.vertx_3_4.server.VertxVersionMatcher.PARSABLE_HEADER_VALUE;
+import static datadog.trace.instrumentation.vertx_3_4.server.VertxVersionMatcher.VIRTUAL_HOST_HANDLER;
+import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.agent.tooling.muzzle.Reference;
+import datadog.trace.api.iast.InstrumentationBridge;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(Instrumenter.class)
+public class HttpServerResponseInstrumentation extends Instrumenter.Iast
+    implements Instrumenter.ForTypeHierarchy {
+  @Override
+  public Reference[] additionalMuzzleReferences() {
+    return new Reference[] {PARSABLE_HEADER_VALUE, VIRTUAL_HOST_HANDLER};
+  }
+
+  public HttpServerResponseInstrumentation() {
+    super("vertx", "vertx-3.4", "Http");
```

Comment:
```suggestion
    super("vertx", "vertx-3.4");
```

I guess we can just keep this like this atm.

Reply:
applied